build/8-to-9-migration/optaplanner-quarkus3.yaml

```diff
@@ -14,13 +14,21 @@ recipeList:
       configuration: <skip>true</skip>
   - org.openrewrite.maven.ChangePropertyValue:
       key: version.io.quarkus
-      newValue: 3.0.0.CR1
+      newValue: 3.0.0.Final
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: version.org.slf4j
+      newValue: 2.0.6
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: version.ch.qos.logback
+      newValue: 1.4.6
```

Comment:
Starting from `1.4.x`, Logback switches to Jakarta. Using that version with OptaPlanner 8 results in no logger implementation found.